tabs/led_strip.html

```diff
@@ -54,11 +54,11 @@
             <button class="funcClear">Clear selected</button>
             <button class="funcClearAll">Clear ALL</button>
 
-            <div class="section">LED Functions</div>
+            <div class="section" i18n="ledStripFunctionSection">LED Functions</div>
```

Comment:
Thanks for pulling out the UI strings.

I've never tried what happens if you leave the text content of the element in when you use `i18n` (presumably the text is overwritten), but I think it's best to remove the text content, to stop people from updating it (instead of the text in `messages.json`) when they change things.